schema/2.0.0/corpora_schema.md

```diff
@@ -145,13 +145,13 @@ The following ontology dependencies are *pinned* for this version of the schema.
 
 cellxgene requires ENSEMBL gene identifiers to ensure that all datasets it stores measure the same features and can therefore be integrated.
 
-The following gene annotation dependencies are *pinned* for this version of the schema.
+The following gene annotation dependencies are *pinned* for this version of the schema. For multi-species experiments, cells from any organism are allowed as long as orthologs from the following organism annotations are used.
```

Comment:
multi-species -> multi-organism

Reply:
Fixed in 960ab161ad9d956b63075a4f3cca5c60d2badf2c